gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -373,6 +373,24 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-threat-critical">
               The color associated with threats of critical severity.
             </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-threat-unspecified-text">
+              The text color on an unspecified threat level background.
```

Comment:
```suggestion
              The text color for use on a threat level "unspecified" background.
```

and similarly for the others

Reply:
[a331a4e](https://github.com/sonatype/sonatype-react-shared-components/pull/1089/commits/a331a4e05f85447c099fb4d15ad5914087ed48a4)